plugins/processors/converter/README.md

```diff
@@ -62,7 +62,7 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
     boolean = []
     float = []
 
-    ## Optional field to use for converting base64 encoding of IEEE 754 Float32 values
+    ## Optional field to use for converting base64 encoding of IEEE 754 Float32 values 
```

Comment:
Can you please revert this change!

Reply:
That got added by running `make docs`, I will remove it.